rocAL/rocAL/include/api/rocal_api_data_loaders.h

```diff
@@ -167,7 +167,7 @@ extern "C"  RocalImage  ROCAL_API_CALL rocalJpegCOCOFileSourcePartial(RocalConte
                                                             bool is_output,
                                                             bool shuffle = false,
                                                             bool loop = false,
-                                                            RocalImageSizeEvaluationPolicy decode_size_policy = ROCAL_USE_MAX_SIZE,
+                                                            RocalImageSizeEvaluationPolicy decode_size_policy = ROCAL_USE_USER_GIVEN_SIZE_RESTRICTED,
```

Comment:
looks like spacing is off here

Reply:
I am not seeing any spacing issues here in github / my editor